### ISSUE TITLE
How to modify softmax

### ISSUE THREAD

**Issue by szxczyc** [KaiyangZhou/deep-person-reid#342]
hi, kaiyang:
If I want to change softmax to AMsoftmax, etc., what should I do?
Is it only need to modify `torchreid / engine / image / softmax.py`?

**Comment by KaiyangZhou** [comment]
you can add the AMSoftmax loss in `torchreid/losses`

maybe this could help https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/person_reidentification

**Comment by szxczyc** [comment]
Thank you very much!

**Comment by CompliceU** [comment]
@szxczyc I don't know how the effect of this loss function is now
